src/server/pkg/deploy/assets/postgres.go

```diff
@@ -120,6 +130,10 @@ func PostgresDeployment(opts *AssetOpts, hostPath string) *apps.Deployment {
 									Name:      "postgres-storage",
 									MountPath: "/var/lib/postgresql/data",
 								},
+								{
+									Name:      postgresInitVolumeName,
+									MountPath: "/docker-entrypoint-initdb.d",
```

Comment:
Just curiosity, but why the reference to docker in the volume name? Does docker interact with postgres somehow?

Reply:
This is part of the postgres Docker image - this is the well-known path where you can mount setup scripts to run when the database is initialized.